tests/test_csv.yaml

```diff
@@ -1,4 +1,4 @@
 benchmarks:
     '*':
-        test_default_metrics:
+        bench-name:
```

Comment:
I don't quite understand what the change in `hpcbench/driver/benchmark.py` does and how this is affecting the `campaign.yaml` files. Could you please document this a bit?

Reply:
Hello @ohm314 
Thank you for the feedback.
There is nothing much to document in the code, but I commented the commit message 7ee6dc1181e21dafd566a3268a630aafedc11935
Please let me know if things are more clear now.